app/src/main/java/com/it3048/mobiledeviceproject/MeetingDetailsForm.kt

```diff
@@ -9,4 +9,5 @@ import androidx.fragment.app.DialogFragment
 import kotlinx.android.synthetic.main.add_meeting_form.view.*
 
 class MeetingDetailsForm: DialogFragment() {
-}
\ No newline at end of file
+  /**TO DO Develop Duncitonality for this class
```

Comment:
In this case, saying what the group should do is not adding value.  

Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.